renovate.json5

```diff
@@ -21,14 +21,22 @@
       "automerge": false,
     },
     {
-      description: "Android Gradle Plugin is tightly coupled to its android/platform/tools/base dependencies",
+      "description": "Android Gradle Plugin is tightly coupled to its android/platform/tools/base dependencies.",
       "groupName": "Android Tools",
       "matchPackagePrefixes": [
         "com.android.tools:",
         "com.android.tools.build:",
-        // LayoutLib intentionally omitted to be updated independently
+        // LayoutLib intentionally omitted to be updated independently.
       ],
       "automerge": false,
-    }
+    },
+    {
+      "description": "LayoutLib shouldn't auto-merge, because updating it is a complex process, see libs/README.md.",
+      "matchPackagePrefixes": [
+        "app.cash.paparazzi:layoutlib-native-",
```

Comment:
```suggestion
        "app.cash.paparazzi:layoutlib-",
```

Reply:
These are the only usages of versions.layoutlib

https://github.com/cashapp/paparazzi/blob/master/gradle/libs.versions.toml#L48-L52

is this for future proofing?

lgtm, commit from UI if you want to go ahead.

Reply:
Yep, future proofing!